torchtune/models/llama2/__init__.py

```diff
@@ -4,7 +4,7 @@
 # This source code is licensed under the BSD-style license found in the
```

Comment:
All changes in models/ will be discarded following merge of https://github.com/pytorch/torchtune/pull/1160